### PR TITLE
Update PromQL upto Jan-20 (@ <timestamp> addition)

### PR DESCRIPTION
**Pull request by blagojts**

### CODE REVIEW

**Review by Harkishen-Singh — APPROVED** [review]
LGTM! We can merge this after a final look from @cevian

**Review by cevian — APPROVED** [review]
Would like to see some end to end test that the enable features flag is correctly passed down, but won't insist on it. You can merge without another review

**Pull request merged**
